index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var semver = require('semver');
+var execa = require('execa');
+var del = require('del');
+
+var exec = function (cmd, args) {
+	const result = execa.sync(cmd, args);
+
+	if (result.stderr !== '') {
```

Comment:
I think it is supported in `master` since we use `childProcess.spawn` there. In `0.4.0` we're still using `execFile` which doesn't have a `stdio` option.